### PR TITLE
subsection for contributors

### PR DESCRIPTION
**Pull request by Sangeetaaa**
Issue Number #47 
Changes proposed in this pull request :section for contributors
[Live Link](https://iit2017032.github.io/Girls-who-code/)

### CODE REVIEW

**Comment by ritikamotwani** [comment]
The whole point of my PR was to organize the code and put the css part in a separate file so I don't think it should be added back 😄 
Further the contributors can be a new section. Contact need not be removed.

**Comment by Sangeetaaa** [comment]
now i had corrected it

File: index.html [hunk 1]
```diff
@@ -214,7 +218,17 @@ <h3 class="title">Contact Person 3</h3>
 </div>
 <p>Feel Free to Contact us for any help</p>
   </div>
- </section>       
+ </section>  
+<div class="subsection" id="contributors" ><p class="cont"  >CONTRIBUTORS</p>
+<i class="cont1" ><b class="name"> Sneha Mishra</b><br><a href="https://github.com/Tlazypanda" class="link">Github profile</a></i>
+<i class="cont2"><b class="name"> Ananya Agrawal
```
Comment (gauthamzz) [inline index.html]: styling is not consistent.

**Review by gauthamzz — CHANGES_REQUESTED** [review]
This section is too large can you make it smaller. Its just names. Why so much space for it.

**Comment by gauthamzz** [comment]
contributers are not at the center in my laptop.

**Comment by gauthamzz** [comment]
@sangeeta2499 fix the merge conflicts as well. I guess you are good to go.

**Comment by Sangeetaaa** [comment]
is it correct now?

**Comment by gauthamzz** [comment]
update your url, you changed your username

**Comment by Sangeetaaa** [comment]
okk

File: index.html [hunk 2]
```diff
@@ -17,14 +17,22 @@
     <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
     <link href="static/css/bootstrap.min.css" rel="stylesheet">
 
-     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-     <link rel="stylesheet" type="text/css" href="style.css">
+
+    
+     <link rel="stylesheet" type="text/css" href="style.css"
```
Comment (gauthamzz) [inline index.html]: The >is not there

**Comment by gauthamzz** [comment]
Idk what you are doing , send me gh pages link of website and finish this by today or I’ll give this to someone else

**Comment by sangeeta2499** [comment]
live link( https://sangeeta2499.github.io/Girls-who-code/)

**Comment by Sangeetaaa** [comment]
live link(https://sangeeta2499.github.io/Girls-who-code/)

**Comment by gauthamzz** [comment]
See the javascript is overflowing at the bottom of the footer.

File: index.html (hunk 2 above)
Comment (gauthamzz) [reply index.html]: Close the <tag> @sangeeta2499

**Review by gauthamzz — CHANGES_REQUESTED** [review]

**Review by Sangeetaaa — COMMENTED** [review]
closing the tag

**Review by Sangeetaaa — COMMENTED** [review]
correcting the overflowing of javascript

**Comment by Sangeetaaa** [comment]
live link(https://sangeeta2499.github.io/Girls-who-code/)

**Pull request merged**